demos/toolbar/toolbar.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
```

Comment:
Why do you need this new file? Can you attach a screenshot of the demo pages before and after? I'm having trouble understanding why this is necessary.

Reply:
You are correct. I committed it by mistake (used it to test out the different mocks). Sorry for the confusion! Will squash.